### PR TITLE
Add list utxo view

### PR DESCRIPTION
**Pull request by vctt94**
This PR adds a list utxo view.

It can be accessed from send tab -> list utxo

image

And looks right now like this:

image

### CODE REVIEW

File: app/components/views/ListUtxo/ListUtxo.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        className="stakepool-purchase-ticket-input-select"
+        {...{ account, onChange: setAccount }}
+      />
+      <div>
+      <div className={styles.tableHeader}>
+        <div>
+          <T id="listutxo.utxo" m="utxo" />
+        </div>
+        <div>
+          <T id="listutxo.value" m="value" />
```
Comment (jholdstock) [inline app/components/views/ListUtxo/ListUtxo.jsx:44]:
```suggestion
          <T id="listutxo.value" m="Value" />
```

File: app/components/views/ListUtxo/ListUtxo.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { StandalonePage, StandaloneHeader } from "layout";
+import { AccountsSelect } from "inputs";
+import { Balance } from "shared";
+import { useListUtxo } from "./hooks";
+import styles from "./ListUtxo.module.css";
+
+const Header = () => (
+  <StandaloneHeader
+    iconClassName="transactions"
+    title={<T id="listutxo.title" m="List Utxo" />}
```
Comment (jholdstock) [inline app/components/views/ListUtxo/ListUtxo.jsx:14]:
```suggestion
    title={<T id="listutxo.title" m="List UTXOs" />}
```

File: app/components/views/ListUtxo/ListUtxo.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      return;
+    }
+    onListUnspentOutputs(account.value)
+      .then(outputs => setUnspentOutputs(outputs));
+    }, [onListUnspentOutputs, account]
+  );
+
+  return (
+    <StandalonePage header={<Header />}>
+      <Subtitle title={<T id="listutxo.subtitle" m="List Unspent Utxo" />} />
```
Comment (jholdstock) [inline app/components/views/ListUtxo/ListUtxo.jsx:33]:
```suggestion
      <Subtitle title={<T id="listutxo.subtitle" m="List Unspent UTXOs" />} />
```

File: app/components/views/ListUtxo/ListUtxo.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      <AccountsSelect
+        className="stakepool-purchase-ticket-input-select"
+        {...{ account, onChange: setAccount }}
+      />
+      <div>
+      <div className={styles.tableHeader}>
+        <div>
+          <T id="listutxo.utxo" m="utxo" />
```
Comment (jholdstock) [inline app/components/views/ListUtxo/ListUtxo.jsx:41]:
```suggestion
          <T id="listutxo.utxo" m="UTXO" />
```

File: app/components/views/TransactionsPage/SendTab/SendTab.js [hunk 5]
```diff
@@ -30,13 +32,30 @@ export const SendTabHeader = () => {
     />
   );
 };
+
+const ListUtxo = ({ onShowListUtxo }) => (
+  <div
+    className={styles.listUtxoLink}
+    onClick={onShowListUtxo}
+  >
+    <T id="transactions.send.listutxo" m="List utxo" />
```
Comment (jholdstock) [inline app/components/views/TransactionsPage/SendTab/SendTab.js:41]:
```suggestion
    <T id="transactions.send.listutxo" m="List UTXOs" />
```

**Review by jholdstock — COMMENTED** [review]
This is great, works well for me. Just some nits on capital letters

**Comment by vctt94** [comment]
Thanks for the review jhold. Updated

**Review by jholdstock — APPROVED** [review]

File: app/components/views/ListUtxo/ListUtxo.jsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    iconClassName="transactions"
+    title={<T id="listutxo.title" m="List UTXOs" />}
+  />
+);
+
+export default () => {
+  const { onListUnspentOutputs } = useListUtxo();
+  const [unspentOutputs, setUnspentOutputs] = useState(null);
+  const [account, setAccount] = useState(null);
+  useEffect(() => {
```
Comment (amassarwi) [inline app/components/views/ListUtxo/ListUtxo.jsx:22]: you could move this with the `useState` to the custom hook

File: app/components/views/ListUtxo/ListUtxo.module.css [hunk 7]
```diff
@@ -0,0 +1,14 @@
+.tableHeader {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  padding: 16px 20px;
+  background-color: var(--sidebar-color);
+}
+
+.utxoTable {
+  display: grid !important;
```
Comment (amassarwi) [inline app/components/views/ListUtxo/ListUtxo.module.css:9]: is the important intended ?

**Review by amassarwi — COMMENTED** [review]
code lgtm, added couple of tiny suggestions

**Review by alexlyp — APPROVED** [review]

**Pull request merged**